lighthouse-core/gather/gatherers/anchor-elements.js

```diff
@@ -47,6 +47,9 @@ function collectAnchorElements() {
     if (node instanceof HTMLAnchorElement) {
       return {
         href: node.href,
+        rawHref: node.getAttribute('href') || '',
+        onclick: node.getAttribute('onclick') || '',
```

Comment:
I'm still a little nervous about pathological sites with ridiculously long onclick handlers. I imagine they have to be out there :) @patrickhulce do you think the string length limit you mentioned before for a different thing would be ok to add to these?

(and if someday we need `onclick` to always be parseable js (or whatever) so it can't be truncated we can always lift the restriction then)

Reply:
I can't imagine the risk is *anywhere* near what a `addEventListener` handler is though.

I'm fine with a `.slice(0, 1024)` or whatever to err on the side of caution though if you think this is a legit issue 🤷

Reply:
> I'm fine with a `.slice(0, 1024)` or whatever to err on the side of caution though if you think this is a legit issue 🤷

I would feel better, yes :) It's a public artifact and if it turns out badly we can't walk it back so easily.

Reply:
Now doing a `onclick.slice(0, 1024)`